FrameworkSystem/private/standardLogging/LoggerWrapper.py

```diff
@@ -0,0 +1,208 @@
+  """
+
+  __gLogging = LoggingWrapper()
+
+  def __init__(self, fathername='', name=''):
+    if fathername == '':
+      self.__logger = logging.getLogger(name)
+    else: 
+      self.__logger = logging.getLogger(fathername).getChild(name)
+
+  def initialized(self):
+    logging.warn("Initialized: Deleted method.")
+    return True
+
+  def showHeaders(self, yesno=True):
+    """
+    Depending on the value, display or not the prefix of the message.
+    :params yesno: boolean determining the behaviour of the display
+    """
+    LoggerWrapper.__gLogging.showHeaders(yesno)
+
+  def showThreadIDs(self, yesno=True):
+    """
+    Depending on the value, display or not the thread ID.
+    :params yesno: boolean determining the behaviour of the display
+    """
+    LoggerWrapper.__gLogging.showThreadIDs(yesno)
+
+  def registerBackends(self, desiredBackends):
+    logging.warn("registerBackends: Deleted method. Logging register its backends itself.")
+
+  def initialize(self, systemName, cfgPath):
+    """
+    Configure gLogger depending on the component used.
+    :params systemName: string represented as "system name/component name"
+    :params cfgPath: string of the cfg file path
+    """
+    LoggerWrapper.__gLogging.loadConfigurationFromCFGFile(systemName, cfgPath)
+
+  def setLevel(self, levelName):
+    """
+    Set a level to the logger.
+    :params levelName: string representing the level to give to the logger
+
+    :return: boolean representing if the setting is done or not
+    """
+    result = False
+    levelName = levelName.upper()
+    if levelName in LogLevels.getLevelNames():
+      self.__logger.setLevel(LogLevels.getLevelValue(levelName))
+      result = True
+    return result
+
+  def getLevel(self):
+    """
+    :return: the name of the level
+    """
+    return LogLevels.getLevel(self.__logger.getEffectiveLevel())
+
+  def shown(self, levelName):
+    """
+    Determine if messages with a certain level will be displayed or not.
+    :params levelName: string representing the level to analyse
+    :return: boolean which give the answer
+    """
+    result = False
+    levelName = levelName.upper()
+    if levelName in LogLevels.getLevelNames():
+      result = self.__logger.isEnabledFor(LogLevels.getLevelValue(levelName))
+    return result
+
+  def getName(self):
+    """
+    :return: "system name/component name"
+    """
+    return LoggerWrapper.__gLogging.getComponent()
+
+  def getSubName(self):
+    """
+    :return: the name of the logger
+    """
+    name = self.__logger.name
+    if name == "root":
+      name = ""
+    else:
+      if name.startswith("root."):
+        name = name.replace("root.", "")
+
+    return name
+
+  def getAllPossibleLevels(self):
+    """
+    :return: a list of all levels available
+    """
+    return LogLevels.getLevelNames()
+
+  def always(self, sMsg, sVarMsg=''):
+    """
+    Always level
+    """
+    level = LogLevels.getLevelValue('ALWAYS')
```

Comment:
Yes, it would be nice but there are many use of string level in the existing code: 
about 110 calls to setLevel in DIRAC and 57 in lhcbDIRAC, so it is complicated.